platformio.ini

```diff
@@ -21,35 +21,44 @@
 env_default = emonesp
 data_dir = src/data
 
+[common]
+version = -DBUILD_TAG=2.0.0
+#MQTT: https://github.com/knolleary/pubsubclient
+lib_deps = PubSubClient@2.6
+
 [env:emonesp]
-platform = espressif
+platform = espressif8266
```

Comment:
is the change `espressif > espressif8266` due to a platformIO change? Is there any difference? I can only see espressif8266 listed on the platformio platform page so I'm assuming it's just been renamed? http://platformio.org/platforms/espressif8266. But `espressif` still seems to work, it's probably a good idea to update my other projects that use this platform.